aggregator/aggregator.go

```diff
@@ -16,7 +16,7 @@ import (
 
 const (
 	AggregatingService = "aggregator"
-	COMMITMENT_SIZE    = 32
+	COMMITMENT_SIZE    = 1
```

Comment:
Just to be able to test sending a batch with 1tx. This will be ultimately fixed by [this](https://github.com/thehubbleproject/hubble-commander/issues/62) issue.